site/app/templates/admin/admin_gradeable/AdminGradeablePeers.twig

```diff
@@ -169,8 +183,134 @@
         });
     }
 
+    function addGraderPeerForm() {
+        // make add grader popup appear
+        $('.popup-form').css('display', 'none');
+        let form = $("#admin-gradeable-peers-form2");
+        form.css("display", "block");
+        $('#admin-gradeable-peers2-submit').prop('disabled', true);
+        captureTabInModal("admin-gradeable-peers-form2");
+        // set the left panel to allow a new grader to be added
+        let peer_grader_pairs = JSON.parse($('#peer_grader_pairs').val());
+        let add_grader_div = $("#new-grader");
+        add_grader_div.empty();
+        add_grader_div.append("Grader to Add:<br />")
```

Comment:
It looks like a lot JS/JQuery is used to build this form, is this something that needs to be dynamically created? or would it be possible to write some of the html in the generic twig form template or its own template file and have it hidden and then display it on the page when needed?